docs/01-app/02-guides/internationalization.mdx

```diff
@@ -18,29 +18,47 @@ It’s recommended to use the user’s language preferences in the browser to se
 
 For example, using the following libraries, you can look at an incoming `Request` to determine which locale to select, based on the `Headers`, locales you plan to support, and the default locale.
 
+```bash filename="Terminal"
+npm install @formatjs/intl-localematcher negotiator
+npm install --save-dev @types/negotiator
+```
+
 ```js filename="proxy.js"
 import { match } from '@formatjs/intl-localematcher'
 import Negotiator from 'negotiator'
 
-let headers = { 'accept-language': 'en-US,en;q=0.5' }
-let languages = new Negotiator({ headers }).languages()
-let locales = ['en-US', 'nl-NL', 'nl']
-let defaultLocale = 'en-US'
+const headers = { 'accept-language': 'en-US,en;q=0.5' }
```

Comment:
My point was that this snippet is not helpful. Here we should've definite the getLocale function, with a comment that shows what headers look like w/ accept-language and then return the match result